### PR TITLE
REST API: Entities API - Load Entities - More Examples on Sorting, Pagination & Fetch Plans

### PR DESCRIPTION
**Pull request by mariodavid**
This PR contains:

* Filter Operator Table improvements
* new dedicated example for fetch plans when loading by ID
* new dedicated examples for sorting
* new dedicated example for pagination

✔️ Grammarly check performed 

## Filter Operator

Bildschirmfoto 2021-03-10 um 12 15 43

## Load Entities Subsection TOC
Bildschirmfoto 2021-03-10 um 12 17 02

### CODE REVIEW

File: content/modules/rest/pages/entities-api/load-entities.adoc [hunk 1]
```diff
@@ -274,36 +292,22 @@ property:: the entity attribute that is being filtered on (like `amount` on the
 +
 In case the attribute is a reference to another entity, it can also be a property path like `customer.name`
 
-operator:: the filter operator. The following values are possible (depending on the property datatype):
+operator:: the filter operator. An operator describes how to filter for a particular attribute. There are multiple operators that can be used independent of the datatype:
```
Comment (knstvk) [inline content/modules/rest/pages/entities-api/load-entities.adoc]: "independently"
BTW, it is suggested by Grammarly

File: content/modules/rest/pages/entities-api/load-entities.adoc [hunk 2]
```diff
@@ -148,119 +171,114 @@ sort:: an entity attribute that will be used for sorting `(String)`.
...
+    "lastModifiedBy": "admin"
+  },
+  {
+    "id": "1068c217-5868-faf4-16aa-23655e9492da",
+    "date": "2021-03-01",
+    "amount": 130.08
   }
 ]
 ----
+<1> the result with the oldest date is returned first
```
Comment (knstvk) [inline content/modules/rest/pages/entities-api/load-entities.adoc]: Start from capital letter, end with dot.

File: content/modules/rest/pages/entities-api/load-entities.adoc [hunk 3]
```diff
@@ -148,119 +171,114 @@ sort:: an entity attribute that will be used for sorting `(String)`.
...
+<2> when the `date` attribute is the same, the `amount` is used to sort the results
 
-NOTE: Every entity in the response has a `_entityName` attribute with the name of the entity, and an `_instanceName` attribute with the xref:data-model:entities.adoc#instance-name[Instance Name] of the entity.
+=== Using Pagination
+
+The Entities API support Pagination to respect the data processing limitation that might be present on the server or client-side. In case you want to load only a particular subset of the entities, you can provide the URL parameters `offset` and `limit`.
```
Comment (knstvk) [inline content/modules/rest/pages/entities-api/load-entities.adoc]: "supports"

Also, about "you can provide the URL parameters `offset` and `limit`": you consistently use similar word composition, but it sounds not very good (not only my opinion, I've validated it).

It should be either with colon: "you can provide the URL parameters: `offset` and `limit`", 
or better in different order: "you can provide the `offset` and `limit` URL parameters".

Please keep it in mind in further writing.

**Comment by knstvk** [comment]
Looks good!
Please correct the mistakes above and merge.

**Review by knstvk — APPROVED** [review]
Looks good! 
Please correct the mistakes above and merge.

File: content/modules/rest/pages/entities-api/load-entities.adoc (hunk 3 above)
Comment (mariodavid) [reply content/modules/rest/pages/entities-api/load-entities.adoc]: I'm not 100% on what you mean by similar word compositions. Is it about this very example or more in general?

I have changed the order accordingly. Will merge it for now. If you have further examples I can try to change it down the road.

**Pull request merged**